### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by njmulsqb**

### CODE REVIEW

File: contributors/index.md [hunk 1]
```diff
@@ -85,6 +85,8 @@ Helmet has help from a lot of contributors. Thanks to everyone on this list!
     - [helmet#345](https://github.com/helmetjs/helmet/pull/345)
   - [Mo Binni](https://twitter.com/mobinni)
     - [helmet#119](https://github.com/helmetjs/helmet/pull/119)
+  - [Najam Ul Saqib](https://njmulsqb.github.io)
+    - [lintJS](https://github.com/helmetjs/helmet/pull/354)
```
Comment (EvanHahn) [inline contributors/index.md]: Could you change this to match the others?

```suggestion
    - [helmet#354](https://github.com/helmetjs/helmet/pull/354)
```

**Review by EvanHahn — CHANGES_REQUESTED** [review]
LGTM other than one small thing!

File: contributors/index.md (hunk 1 above)
Comment (njmulsqb) [reply contributors/index.md]: Done

**Review by EvanHahn — APPROVED** [review]
Thanks again!

**Pull request merged**